sources/packages/web/src/views/student/StudentDashboard.vue

```diff
@@ -1,12 +1,15 @@
 <template>
-  <full-page-container>
+  <v-container>
     <RestrictionBanner
       v-if="hasRestriction"
       :restrictionMessage="restrictionMessage"
     />
     <CheckValidSINBanner />
-    <formio formName="studentdashboard"></formio>
-  </full-page-container>
+    <formio
+      formName="studentwelcomepage"
+      @submitted="goToStudentApplication"
```

Comment:
Please avoid using the submit for this case. We have examples of other buttons executing a redirect.
Please take a look on below file to check how _customEvent_ were used.
https://github.com/bcgov/SIMS/blob/8881a35e353ebfb82f01dda7012e33ebaf86cbaa/sources/packages/web/src/components/students/ApplicationDetails.vue